### DISCUSSION TITLE
Exposing custom directory as "static"

### DISCUSSION THREAD

**Discussion by msurdi** [nuxt/nuxt#9520] [Questions (Nuxt 2)] [answered]
Hi!

Let's say I have a package, which once installed in my project, as a bunch of static files (such as images, etc) at `./node_modules/my-package/dist/images/**`.

Is there any way to map the url `http://localhost:3000/images/**`  (or similar) to the filesystem path mentioned above?

Probably this can be done with a custom server, but this project does not need a server so I'm looking for better alternatives than just doing a `cp -r` during build.

Thanks!

**Comment by TheAlexLichter** [comment, marked as answer]
You can use a nuxt hook to copy the files into the /static output dir during the generation ☺️

This also happens in the [sitemap-module](https://github.com/nuxt-community/sitemap-module/blob/dev/lib/generator.js#L58) when using SSG for example 😋

**Reply by msurdi** [reply]
Great!. Thanks a lot.
